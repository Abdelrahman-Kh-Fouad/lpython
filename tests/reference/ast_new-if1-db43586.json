{
    "basename": "ast_new-if1-db43586",
    "cmd": "lpython --show-ast --new-parser --no-color {infile} -o {outfile}",
    "infile": "tests/parser/if1.py",
    "infile_hash": "98cfe7d93a86144b5da023ee9f46656e0c59e97bc004164f6b859e15",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_new-if1-db43586.stdout",
    "stdout_hash": "f8b8b5560efc232d9b97deda39057d25ddd5631e1863045d16015784",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}