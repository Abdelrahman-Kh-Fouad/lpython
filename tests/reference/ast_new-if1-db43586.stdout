(Module [(Assign [(Tuple [(Name x Store)] Store)] (ConstantInt 10 ()) ()) (Assign [(Tuple [(Name y Store)] Store)] (ConstantInt 14 ()) ()) (Assign [(Tuple [(Name z Store)] Store)] (ConstantInt 12 ()) ()) (If (BoolOp And [(Compare (Name x Load) GtE [(Name y Load)]) (Compare (Name x Load) GtE [(Name z Load)])]) [(Assign [(Tuple [(Name largest Store)] Store)] (Name x Load) ()) (Expr (Call (Name print Load) [(ConstantStr "Largest: " ()) (Name x Load)] []))] [(If (BoolOp And [(Compare (Name y Load) GtE [(Name x Load)]) (Compare (Name y Load) GtE [(Name z Load)])]) [(Assign [(Tuple [(Name largest Store)] Store)] (Name y Load) ()) (Expr (Call (Name print Load) [(ConstantStr "Largest: " ()) (Name y Load)] []))] [(Assign [(Tuple [(Name largest Store)] Store)] (Name z Load) ()) (Expr (Call (Name print Load) [(ConstantStr "Largest: " ()) (Name z Load)] []))])]) (If (Compare (Name y Load) Gt [(ConstantInt 10 ())]) [(Expr (Call (Name print Load) [(ConstantStr "Above ten," ())] [])) (If (Compare (Name y Load) Gt [(ConstantInt 20 ())]) [(Expr (Call (Name print Load) [(ConstantStr "and also above 20!" ())] []))] [(Expr (Call (Name print Load) [(ConstantStr "but not above 20." ())] []))])] [])] [])
